src/components/app/ProfileViewer.js

```diff
@@ -52,7 +53,6 @@ class ProfileViewer extends PureComponent<Props, State> {
   constructor(props) {
     super(props);
     (this: any)._onSelectTab = this._onSelectTab.bind(this);
-    this.state = { isMounted: false };
```

Comment:
nit: please remove the type State above too.
The type declaration should read `extends PureComponent<Props>` and you can just delete the type State above. When we don't specify any state, it defaults to undefined.

Reply:
Oops, sorry, missed that completely! It is fixed now